tests/ui_tests/app/configuration.py

```diff
@@ -55,16 +68,40 @@ def assert_field_is_editable(self, field_element, editable=True) -> None:
             self.is_element_editable(field_element) == editable
         ), f"Field {field_element} should {'be editable' if editable else 'not be editable'}"
 
+    def get_api_value(self, adcm_credentials, app_fs, field: str):
+        """Gets field value by api"""
+
+        current_config = _make_request(
+            adcm_credentials, app_fs, "GET", f"/api/v1/{self.driver.current_url.replace(app_fs.adcm.url, '')}/current"
+        ).text
```

Comment:
why not `.json()`?

Reply:
why json?

Reply:
why json?

Reply:
as I understand the next thing you do, you load json from text to present it as python dict, no?

Reply:
changed getting config